### PR TITLE
Setup nginx on app3 for media serving on the streamwebs websites

### PR DESCRIPTION
**Pull request by ramereth**
The media folder on Django requires a webserver to serve them and this basically
implements it for the streamwebs production and staging websites. It sets it up
so that /<app name>/<whatever dir you need> is redirected properly from HAproxy.

So HAProxy knows that when it connects to this server, it needs to access
/streamwebs-production/media when a user requests /media.

This is related to osuosl/streamwebs#443.

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+default['osl-app']['nginx'] = {}
+default['osl-app']['haproxy_ips'] = %w(140.211.9.53 140.211.9.52 140.211.9.50)
```
Comment (doublej472) [inline attributes/default.rb]: Hardcoding IP's in the attributes file?

Comment (ramereth) [reply attributes/default.rb]: We don't have an easy way to pull this in unfortunately.

**Comment by doublej472** [comment]
I'll try again in case it's just some temporary networking issue.

**Comment by ramereth** [comment]
!bump patch *

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v1.11.11.  Have a nice day!

**Pull request merged**